vividus-plugin-mongodb/src/main/resources/spring.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:util="http://www.springframework.org/schema/util"
+       xsi:schemaLocation="
+        http://www.springframework.org/schema/beans https://www.springframework.org/schema/beans/spring-beans.xsd
+        http://www.springframework.org/schema/util https://www.springframework.org/schema/util/spring-util.xsd"
+       default-lazy-init="true">
+
+    <bean id="mongoDbSteps" class="org.vividus.bdd.steps.mongodb.MongoDbSteps">
+        <constructor-arg index="0">
+            <bean factory-bean="propertyParser" factory-method="getPropertyValuesByPrefix">
+                <constructor-arg value="mongodb.connection." />
+            </bean>
+        </constructor-arg>
+        <constructor-arg index="1" ref="jsonUtils-LowerCamelCase" />
+    </bean>
+
+    <bean class="org.vividus.bdd.converter.StringToBsonConverter" />
```

Comment:
Use `@Named` instead of explicit declaration to keep consistency with other modules